spec/forms/decidim/vocdoni/admin/setup_form_spec.rb

```diff
@@ -24,13 +24,15 @@
       hash_including({
                        minimum_answers: "Each question has <strong>at least 2 answers</strong>.",
                        minimum_questions: "The election has <strong>at least 1 question</strong>.",
-                       published: "The election is <strong>published</strong>."
+                       published: "The election is <strong>published</strong>.",
+                       time_before: "The setup is being done <strong>at least 10 minutes</strong> before the election starts.",
```

Comment:
Note: we should use interpolation here with the value of the configured variable `setup_minimum_minutes_before_start`